src/models/indexable.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Model for the Indexable table.
+ *
+ * @package Yoast\YoastSEO\Models
+ */
+
+namespace Yoast\YoastSEO\Models;
+
+use Yoast\YoastSEO\Yoast_Model;
+
+/**
+ * Indexable table definition.
+ *
+ * @property int     $id
+ * @property int     $object_id
+ * @property string  $object_type
+ * @property string  $object_sub_type
+ *
+ * @property string  $created_at
+ * @property string  $updated_at
+ *
+ * @property string  $permalink
+ * @property string  $canonical
+ * @property int     $content_score
+ *
+ * @property boolean $is_robots_noindex
+ * @property boolean $is_robots_nofollow
+ * @property boolean $is_robots_noarchive
+ * @property boolean $is_robots_noimageindex
+ * @property boolean $is_robots_nosnippet
+ *
+ * @property string  $title
+ * @property string  $description
+ * @property string  $breadcrumb_title
+ *
+ * @property boolean $is_cornerstone
+ *
+ * @property string  $og_title
```

Comment:
Looking at the migration these seem no longer part of the Indexable table definition.